scripts/bluetooth/mesh/mesh_dfu_metadata.py

```diff
@@ -34,7 +34,6 @@
 def exit_with_error_msg():
     traceback.print_exc()
     print("Extracting BLE Mesh metadata failed")
-    print("You can bypass this script by disabling the CONFIG_BT_MESH_DFU_METADATA_ON_BUILD option in your project config")
```

Comment:
We want to keep this option.

Reply:
As above, this option does nothing and has been unused since the migration to sysbuild